.github/workflows/e2e-tests.yml

```diff
@@ -23,43 +23,16 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ inputs.branch }}
-      - name: Set up Python ${{inputs.python-version}}
-        uses: actions/setup-python@v5
+      - name: Install the latest version of uv with Python ${{inputs.python-version}}
```

Comment:
should I update this branch `docs-1.0.0` to be in sync with the `main`. It's currently missing this changes https://github.com/kedro-org/kedro/pull/5003 

@ankatiyar @ElenaKhaustova ?